CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+Contributing to Emperor
+=======================
+
+[Emperor](http://emperor.colorado.edu) is an open source software package, and we welcome community contributions. You can find the source code and test code for Emperor under public revision control in the Emperor git repository on [GitHub](https://github.com/biocore/emperor). We very much welcome contributions.
```

Comment:
Probably, other thing is that we could just use the GitHub URL i.e.
http://biocore.github.io/emperor/. I don't have strong feelings though.

On (Feb-22-15| 9:54), Antonio Gonzalez wrote:
> > @@ -0,0 +1,122 @@
> > +Contributing to Emperor
> > +=======================
> > +
> > +[Emperor](http://emperor.colorado.edu) is an open source software package, and we welcome community contributions. You can find the source code and test code for Emperor under public revision control in the Emperor git repository on [GitHub](https://github.com/biocore/emperor). We very much welcome contributions.
> 
> Should we create emperor.microbio.me? 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/348/files#r25134469